### DISCUSSION TITLE
interesting!  questions!

### DISCUSSION THREAD

**Discussion by rhaynes74** [epilys/bibliothecula#3] [General]
Hi folks, I hope all is well.  Are there some install instructions?

**Comment by epilys** [comment]
Hello! There are various tools in this repository, are you referring to something in particular? The core database schema is just SQL statements you can execute in an [`sqlite` database](https://www.sqlite.org/) which is included in most if not all Linux distributions and also available for Windows and OSX.

I will write install instructions for the tools and also for setting up a database. If you want to try a specific tool I can prioritize it now, just let me know.

**Comment by rhaynes74** [comment]
Hi Manos  - I think at first a screencast of a workflow would be good. I am interested in all tools!

Sincerely,

Dr. Ronald D. Haynes
Professor, Department of Mathematics and Statistics
Chair, MSc and Phd Scientific Computing Programs
Memorial University of Newfoundland

We acknowledge that the lands on which Memorial University’s campuses are situated are in the traditional territories of diverse Indigenous groups, and we acknowledge with respect the diverse histories and cultures of the Beothuk, Mi’kmaq, Innu, and Inuit of this province.

**Comment by rhaynes74** [comment]
What are the next steps after creation of the database and tables?
